core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -3931,6 +3939,34 @@ public static List distinct(List list) {
     return min;
   }
 
+  /** Support the ARRAY_PREPEND function. */
+  public static List arrayPrepend(List list, Object element) {
+    final List result = new ArrayList(list.size() + 1);
+    result.add(element);
+    result.addAll(list);
+    return result;
+  }
+
+  /** Support the ARRAY_POSITION function. */
+  public static Long arrayPosition(List list, Object element) {
+    final int index = list.indexOf(element);
+    if (index != -1) {
```

Comment:
Can you resolve it?
![image](https://github.com/apache/calcite/assets/45968640/e6cc7524-0d5b-4f47-8f58-3ce0a66a15f1)

Reply:
fixed @JiajunBernoulli